docs/concepts.md

```diff
@@ -24,54 +24,40 @@ For output, being incremental means that Feldera outputs query results
 as sets of changes from the previous output. This improves performance
 and reduces output size.
 
-## Programs
+## Pipeline
 
-In Feldera, a SQL **program** is a collection of SQL DDL (table and
-view definitions):
+A Feldera **pipeline** pulls data from sources, runs it through a
+computation, and pushes the results to sinks.  The user can start,
+stop, manage, and monitor pipelines.
 
```

Comment:
It might help to add a sentence to the effect of "A pipeline is created by attaching input and output [connectors](#connector) to a SQL [program](#program)".

Reply:
Done, thanks.